decidim-proposals/app/assets/javascripts/decidim/proposals/admin/proposals.js.erb

```diff
@@ -0,0 +1,112 @@
+// = require_self
+
+let selectedProposalsCount = function() {
```

Comment:
If you don't need ERB tags here then move it to ES6 syntax (change the `erb` extension to `es6`).

Also, shouldn't this be wrapped in a `$(){ ... }` block? 😕 (JS is not my forte)

Reply:
Yeah, all this should probably be run on document initialization time, and enclosed in a closure. So this seems like a good option:

```javascript
$(function(){

});
```

If you want a function to be called from the outside, register it into window:

```javascript
window.whatever = function(){

}
```

Reply:
Ok, I'll rename it to `es6`.

Yes it should be wrapped as you said, I'll fix it